trajectory_msgs/JointTrajectoryPoint Pose
---
bool Collision